src/modules/Security.rb

```diff
@@ -726,6 +729,19 @@ def Write
     # @return [Boolean] True on success
     def Import(settings)
       settings = deep_copy(settings)
+      if settings.key?("KERNEL.SYSRQ")
```

Comment:
Why are you fixing the 'case' of these attributes only? What about the rest? On the other hand, the hash will have some "duplicated" values:

```ruby
{
  "KERNEL.SYSRQ" => "1",
  "kernel.sysrq" => "1"
}
```

It will work, but it might be confusing. You can avoid this by using: `#delete`:

```ruby
settings["kernel.sysrq"] = settings.delete("KERNEL.SYSRQ")
```